main.js

```diff
@@ -564,6 +571,23 @@ app.on('ready', function() {
       require('open')('mailto:project@openbazaar.org?subject=OpenBazaar Debug Report&body=' + body);
 
     }},
+    {label: 'View Python Error Log', type: 'normal', click: function() {
```

Comment:
Weird. Master should be free of es-lint errors. I don't know why you had some and then they disappeared.